wear/src/main/java/io/homeassistant/companion/android/sensors/HealthServicesSensorManager.kt

```diff
@@ -48,8 +71,20 @@ class HealthServicesSensorManager : SensorManager {
     override val name: Int
         get() = commonR.string.sensor_name_health_services
 
-    override fun getAvailableSensors(context: Context): List<SensorManager.BasicSensor> {
-        return listOf(userActivityState)
+    override suspend fun getAvailableSensors(context: Context, intent: Intent?): List<SensorManager.BasicSensor> {
+        latestContext = context
+        if (healthClient == null)
+            healthClient = HealthServices.getClient(latestContext)
+        if (passiveMonitoringClient == null)
+            passiveMonitoringClient = healthClient?.passiveMonitoringClient
+        if (passiveMonitoringCapabilities == null)
+            passiveMonitoringCapabilities = passiveMonitoringClient?.getCapabilitiesAsync()?.await()
+
+        val supportedSensors = mutableListOf(userActivityState)
+
+        if (passiveMonitoringCapabilities?.supportedDataTypesPassiveMonitoring?.contains(DataType.FLOORS_DAILY) == true)
+            supportedSensors += supportedSensors.plus(dailyFloors)

```

Comment:
Doesn't this create duplicates? I think this means the existing list which has `userActivityState` _plus [`userActivityState`, `dailyFloors`]_.

```suggestion
            supportedSensors += dailyFloors
```

Reply:
Accidentally missed this one earlier when I fixed a similar issue down below :)